StPetersburg/InEtn40559.xml

```diff
@@ -0,0 +1,224 @@
+                    Bet, Gimel, Dalet, He, Vav.</note>
+                  </msItem>
+                  <msItem xml:id="ms_i3">
+                    <locus from="1rb" n="1"></locus>
+                    <locus to="1rb" n="35"></locus>
+                    <title type="complete" ref="LIT5847Asmat"><certainty locus="value" match="../@ref" cert="medium"></certainty></title>
+                    <textLang mainLang="gez"></textLang>
+                    <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡
+                    ፩ አምላክ፡ ወትብል፡ ቃል፡ ጸሎተ፡ አስማተ፡ ዓስማተ፡</incipit>
+                  </msItem>
+                  <msItem xml:id="ms_i4">
+                    <locus from="1rb" n="36"></locus>
+                    <locus from="1rb" n="115"></locus>
+                    <title type="complete" ref="LIT2765RepCh49"></title>
+                    <textLang mainLang="gez"></textLang>
+                    <incipit xml:lang="gez">ሰላም፡ ለከ፡ ሰዳዴ፡ ሰይጣናት፡ ፋኑኤል፡ ለእግዚአብሔር፡
+                    እምጽርሑ፡ ከመ፡ ኢይስክዩ፡ ሰብአ፡ እለ፡ ይኔስሑ፡</incipit>
+                  </msItem>
+                  <msItem xml:id="ms_i5">
+                    <locus from="2r" n="1"></locus>
+                    <locus to="2r" n="68"></locus>
+                    <title type="complete" ref="LIT5876AsmatPrayer"></title>
+                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"></gap> ኤኮስ፡ አስሌ፡ ኤፓ፡</incipit>
+                  </msItem>
+                  <msItem xml:id="ms_i6">
+                    <locus from="2r" n="69"></locus>
+                    <locus to="2r" n="85"></locus>
+                    <title type="incomplete" ref="LIT2452Tergwa"><certainty locus="value" match="../@ref" cert="medium"></certainty></title>
+                    <textLang mainLang="gez"></textLang>
+                    <incipit xml:lang="gez">አሌፍ፡ ብፁዓን፡ እለ፡ ንፁሃን፡</incipit>
+                  </msItem>
+                </msContents>
+                <physDesc>
+                  <objectDesc form="Scroll">
+                      <supportDesc>
+                          <support>
+                              <material key="parchment"/>
+                              <note>A parchment strip of 14,5 cm is attached to the top of the scroll.</note>
+                          </support>
+                          <extent>
+                              <measure unit="leaf">2</measure>
+                              <dimensions type="outer" unit="mm">
+                                <height>1445</height>
+                                <width>75</width>
+                              </dimensions>
+                          </extent>
+                          <condition key="deficient">The parchement was torn and repaired with a piece of white thread.</condition>
+                      </supportDesc>
+                      <layoutDesc>
+                        <layout>First 14 lines are written in 1 column, further the text is layouted in 2 columns
```

Comment:
I think the use of different `<layout>` elements for each unit of layout, as you have done, is exactly right. I would further try to separate the information into each layout element, with reference to their place in the scroll and the strip on which they are contained. I am aware that the guidelines on all this are suboptimal and it all needs reworking. If you want to close this PR before the scroll issue is resolved, I think separating the information into each <layout> would be good, and could then be later corrected once we have better guidelines (this is of course true for all existing scroll descriptions, not only these!)

Reply:
done